tests/framework/ROM/TimeSeries/SyntheticHistory/TrainingData/Interpolated_A.csv

```diff
@@ -0,0 +1,3 @@
+macro,scaling,filename
+0,1,Clustered_A_0.csv
+1,1,Clustered_A_0.csv
```

Comment:
Is this testing a skipped year? From `0` to `2` maybe?

Reply:
It is not, thanks for catching that! I reverted it back to reproduce the error when I submitted the Github issue and forgot to change it back. Will fix now